kodilla-spring/src/main/java/com/kodilla/spring/portfolio/BoardConfig.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.spring.portfolio;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Scope;
+
+@Configuration
+public class BoardConfig {
+
+    @Autowired
+    @Qualifier("toDoList")
+    TaskList taskList;
```

Comment:
czemu taki sierota? Jeśli wstrzykujesz w to pole beana toDoList, to w linii 20 nie musisz na nowo wolac metody, tylko podac tego beana